webui/webui/components/chat.py

```diff
@@ -14,23 +16,23 @@ def message(qa: QA) -> rx.Component:
     Returns:
         A component displaying the question/answer pair.
     """
-    return rx.chakra.box(
-        rx.chakra.box(
+    return rx.box(
+        rx.box(
             rx.chakra.text(
                 qa.question,
-                bg=styles.border_color,
-                shadow=styles.shadow_light,
-                **styles.message_style,
+                background_color=rx.color("mauve", 4),
+                color=rx.color("mauve", 12),
+                **message_style,
             ),
             text_align="right",
             margin_top="1em",
         ),
-        rx.chakra.box(
+        rx.box(
             rx.chakra.text(
                 qa.answer,
-                bg=styles.accent_color,
-                shadow=styles.shadow_light,
-                **styles.message_style,
+                background_color=rx.color("accent", 4),
```

Comment:
Can we just use `color_scheme` here

Reply:
Can't do that here because text colors schemes don't have backgrounds